CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ if (EIGEN3_FOUND)
 endif (EIGEN3_FOUND)
 # Boost
 include(FindOrFetchBoost)
+# Cereal
+include(FindOrFetchCereal)
```

Comment:
Should we include `FindOrFetchCatch2` too?

Reply:
We can discuss how to organize unit tests .. right now the only dir with unit tests loads catch2